LICENSE

```diff
@@ -1,6 +1,6 @@
-MIT License
+UNLICENSED License
```

Comment:
We are using MIT license, so this change should be reverted.

Reply:
Fixed